CONTRIBUTING.md

```diff
@@ -7,9 +7,9 @@
 
 ## Setup Development (developed using Node v4.3.1)
 
- - `npm install -g gulp-cli`
- - `npm install -g karma-cli`
```

Comment:
if we remove these global packages then manually running any `gulp` or `karma` command will have to be from the `node_modules` folder.

eg.
`./node_modules/gulp/bin/gulp.js -T`

Notice `gulp.js` is executable. (`ls -la ./node_modules/gulp/bin/gulp.js`)